mainone.js

```diff
@@ -0,0 +1,3 @@
+let cost = 1
+let tax = (cost * 0.1)
+console.log =(cost + tax + "")
```

Comment:
console.log is not a variable, so you should not assign anything to it!
Spacing and semicolon!
```suggestion
console.log(cost + tax + "");
```